google/cloud/forseti/common/util/regular_exp.py

```diff
@@ -32,7 +32,7 @@ def escape_and_globify(pattern_string):
         pattern_string (str): The pattern string of which to make a regex.
 
     Returns:
-        str: The pattern string, escaped except for the "*", which is
+        str: The pattern string, escaped except for the "\*", which is
```

Comment:
Why is this changed to `\*`?  The behavior of the code on line 42, shows that `*` is what's handled?

Reply:
Got a warning on that line and wanted to test to see if that was what it was flagging- changed it back.